goon/browserassets/js/browserOutput.js

```diff
@@ -244,6 +255,14 @@ function highlightTerms(el) {
 			}
 		}
 	}
+
+	// Code phrases
+	for (var i = 0; i < opts.codePhrases.length; i++) {
+		highlightRecursor(el, escapeRegexCharacters(opts.codePhrases[i]), "codephrases");
```

Comment:
My only concern with this is code phrases quite often contain player names. EG: `John Smith` is an online player, and their name will be selected from the codeword pool, and this would make any instance of their name appearing automatically change span, which can be confusing.

Reply:
As long as it doesn't show on their name in the `John Smith says, "Hi"` thing, I think this is fine.

Reply:
All chat messages are treated equal, which has just made me realise something else. If an admin rolls traitor, any logs which can have the target name (or area since that's also valid) are going to be coloured, which may break things, or at the very least, look weird.

Reply:
This issue has been fixed by tweaking the CSS rules.